File: app/src/main/java/langchain4j/AnthropicSky.java
package langchain4j;

import dev.langchain4j.model.chat.ChatModel;
import lombok.extern.slf4j.Slf4j;

@Slf4j
public class AnthropicSky {

  public static void main(String[] args) {
    ChatModel model = ChatModelFactory.getChatModel(ChatModelProvider.ANTHROPIC);

    // Send a user message and get the response
    String response = model.chat("Why is the sky blue?");

    log.info(response);
  }
}

File: app/src/main/kotlin/langchain4j/AnthropicSky.kt
package langchain4j

import dev.langchain4j.model.chat.ChatModel
import mu.KotlinLogging

private val log = KotlinLogging.logger {}

fun main() {
    val model: ChatModel = getChatModel(ChatProvider.ANTHROPIC)

    // Send a user message and get the response
    val response = model.chat("Why is the sky blue?")

    log.info { response }
}
